src/mcts/node.h

```diff
@@ -152,9 +153,13 @@ class Node {
   uint32_t GetChildrenVisits() const { return n_ > 0 ? n_ - 1 : 0; }
   // Returns n = n_if_flight.
   int GetNStarted() const { return n_ + n_in_flight_; }
+  float GetQ(float draw_score) const {
+    if (draw_score) return wl_ + draw_score * d_;
```

Comment:
Why do we need the `if` here? Is it to guard against `d_` being NaN or Inf?

Reply:
I just thought it would be a bit faster in default case that way, but tests shown no difference. Will remove.